hello.py

```diff
@@ -0,0 +1,5 @@
+def hello():
+    print("Hello PyCon APAC 2022")
+
+if __name__ == '__main__':
+    hello()
```

Comment:
Hi @adinpearce , maybe time you want to pay attention to the end of line.

Reply:
Sorry, I will need more specific instructions.

Reply:
![Selection_017](https://user-images.githubusercontent.com/3217432/159274082-06037bcd-e6f9-40a4-92b9-e9071f781ea8.png)

Reply:
It's ok. I fixed that today along with michelle! https://github.com/yuyinwangw/cli_test/pull/2/files#diff-f10daf8f1ee9ded24fd9ddd7ca6a8171bb0888894d9b805a75dac52078bd4cfcL5